recipes/cem/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "0.1.5" %}
+
+package:
+  name: cem
+  version: {{ version }}
+
+source:
+  url: https://github.com/csdms-contrib/cem/archive/v{{ version }}.tar.gz
+  sha256: 8206a4fd880fbfa595c659608c22a84ad528fbd69b4160489a906325ceabdc19
+  patches:
+    - remove_ncurses_h.patch  # Remove unneeded ncurses.h include
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    - pkg-config
+    - {{ compiler('c') }}
+  host:
+    - glib
+  run:
+    - glib
+
+
+test:
+  commands:
+    - cem --version
+    - cem --help
+
+about:
+  home: http://csdms.colorado.edu/wiki/Model:CEM
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
I found two license files in the package. Why there are two?

Reply:
I sent a PR, which has now been merged and a new release created, to fix this and clean up a couple other things. We'll see how the builds go but, I think, things should be in good shape now.

Thanks for all your help, @marcelotrevisani!